### DISCUSSION TITLE
What does FSmooth do in MLABecLaplacian?

### DISCUSSION THREAD

**Discussion by ankithadas** [AMReX-Codes/amrex#3512] [Q&A] [answered]
Hey AMReX team,

I quite didn't understand the purpose of FSmooth in MLABecLaplacian solver. Any explanation regarding the purpose of this function would be much appreciated 

Kind regards,
Ankith

**Comment by WeiqunZhang** [comment, marked as answer]
https://en.wikipedia.org/wiki/Multigrid_method
> Smoothing – reducing high frequency errors, for example using a few [iterations](https://en.wikipedia.org/wiki/Iterative_method) of the [Gauss–Seidel method](https://en.wikipedia.org/wiki/Gauss%E2%80%93Seidel_method).

**Reply by ankithadas** [reply]
Ahh so abec_gsrb does Gauss Seidel iteration using Red Black coloring. Thank you
